<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="MainStackWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>751</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Mpage1">
     <widget class="QLabel" name="UsernamLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="PwdLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LoginButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>290</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Log in</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="UsernameLE">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>120</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="PwdLE">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>190</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Mpage2">
     <widget class="QStackedWidget" name="StudentStackWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>751</width>
        <height>391</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Studpage0">
       <widget class="QLabel" name="StudentWelcomLabel">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>20</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Welcome to the student page</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ViewInfoButton">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>350</y>
          <width>351</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>View Info</string>
        </property>
       </widget>
       <widget class="QPushButton" name="logoutButton">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>20</y>
          <width>71</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>logout</string>
        </property>
       </widget>
       <widget class="QListWidget" name="StudClassListWidget">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>121</y>
          <width>256</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="StudClassListWlabel">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>100</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Select a course</string>
        </property>
       </widget>
       <widget class="QLabel" name="StudNameLabelMP">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Student name</string>
        </property>
       </widget>
       <widget class="QLabel" name="StudentIDlabelMP">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Student ID</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="StudGroupBox">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>99</y>
          <width>361</width>
          <height>221</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Assignments and grades for the selected course</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Studpage1">
       <widget class="QLabel" name="StudInfoLabel">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>40</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Student info page</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Studbackbutton1">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>350</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>back</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Mpage3">
     <widget class="QStackedWidget" name="InstructorStackWidget">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>20</y>
        <width>761</width>
        <height>391</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Instpage0_2">
       <widget class="QListWidget" name="InstClassListWidget">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>100</y>
          <width>256</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="InstStudListWidget">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>100</y>
          <width>256</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="InstNameLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Instructor's Name</string>
        </property>
       </widget>
       <widget class="QLabel" name="InstIDlabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Instructor's ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="InstSelectCourseLabel">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>80</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Select a course</string>
        </property>
       </widget>
       <widget class="QLabel" name="InstStudListLabel">
        <property name="geometry">
         <rect>
          <x>509</x>
          <y>80</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>List of tstudent per class</string>
        </property>
       </widget>
       <widget class="QPushButton" name="InstLogoutButton">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>10</y>
          <width>71</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
       <widget class="QPushButton" name="InstInfoButton">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>350</y>
          <width>231</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>View Info</string>
        </property>
       </widget>
       <widget class="QLabel" name="InstWelcomePageLabel">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>10</y>
          <width>201</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Welcome to the instructor's page</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Instpage1">
       <widget class="QLabel" name="InstructorWelcomLabel">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>20</y>
          <width>201</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Welcome to the Instructors Page</string>
        </property>
       </widget>
       <widget class="QPushButton" name="InstBackButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>350</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Mpage4">
     <widget class="QStackedWidget" name="AdminStackedWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>751</width>
        <height>401</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Admipage0">
       <widget class="QLabel" name="AdminWelcomePageLabel">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>20</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Welcome to the admin page</string>
        </property>
       </widget>
       <widget class="QListWidget" name="AdmStudListWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>211</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="AdmInstListWidget">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>120</y>
          <width>211</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="AdmClassListWidget">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>120</y>
          <width>201</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="AdminLogoutButton">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>10</y>
          <width>71</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
       <widget class="QLabel" name="AdminNameLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Admin's Name</string>
        </property>
       </widget>
       <widget class="QLabel" name="AmdinIDlabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Admin's ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AdmStudVInfoButton">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>350</y>
          <width>101</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>View Info</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AdmStuADDbutton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>320</y>
          <width>111</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AdmStudDELbutton">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>320</y>
          <width>101</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AdmStudVClassButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>350</y>
          <width>111</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>View Classes</string>
        </property>
       </widget>
       <widget class="QLabel" name="AdminStudListLabel">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>90</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Select a Student</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AdmInstADDbutton">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>320</y>
          <width>111</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AdmInstDELbutton">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>320</y>
          <width>101</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AdmInstVClassButton">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>350</y>
          <width>111</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>View Classes</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AdmInstVInfoButton">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>350</y>
          <width>101</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>View Info</string>
        </property>
       </widget>
       <widget class="QLabel" name="AdminInstListLabel">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>90</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Select an Instructor</string>
        </property>
       </widget>
       <widget class="QLabel" name="AmdinClassListLabel">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>100</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Select a class</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AdmClassADDbutton">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>320</y>
          <width>201</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AdmClassDELbutton">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>350</y>
          <width>101</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AdmClassVInfo">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>350</y>
          <width>101</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>View Info</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Admipage1"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>786</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
